spec/models/post_methods_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.create(
+    name: 'Doe', photo: 'https://doe.com/me.png',
+    bio: 'Iam John Doe.', post_counter: 0
+  )
+
+  post = Post.create(
+    title: 'About', text: 'About me', comments_counter: 1,
+    likes_counter: 0, author_id: user.id
+  )
+
+  8.times.collect do
+    Comment.create(post:, author: user, text: 'first comment')
+  end
+
+  context 'recent_five_comments' do
+    it 'return at most 5 comments' do
+      puts post.recent_five_comments.length
```

Comment:
- **[OPTIONAL]** I strongly recommend you remove this `puts` call here.

Reply:
thanks you @DammyShittu  am going to work on it